src/realm/sync/protocol.hpp

```diff
@@ -60,6 +60,9 @@ namespace sync {
 //   13 Support for syncing collections (lists and dictionaries) in Mixed columns and
 //      collections of Mixed
 //
+//   14 Support for server initiated bootstraps, including bootstraps for role/
```

Comment:
I was waiting until this feature was about to be merged, but it really doesn't matter, since I'm using a feature branch and the feature is behind a feature flag.